src/main/java/cms/gov/madie/measure/resources/ExportController.java

```diff
@@ -53,6 +55,7 @@ public ResponseEntity<byte[]> getZip(
         .body(bundleService.exportBundleMeasure(measure, accessToken));
   }
 
+  @Deprecated
```

Comment:
I think we can safely remove this API, as you are already updating the only call to this API in madie-patient.
https://github.com/MeasureAuthoringTool/madie-patient/pull/322/files

Reply:
+1

Reply:
done. yeah I wasn't sure if that part would get done or not